bdsx/bds/packets.ts

```diff
@@ -624,12 +807,34 @@ export class PlaySoundPacket extends Packet {
 
 @nativeClass(null)
 export class StopSoundPacket extends Packet {
-    // unknown
+    @nativeField(CxxString)
+    soundName:CxxString;
+    @nativeField(bool_t)
+    stopAll:bool_t;
 }
 
 @nativeClass(null)
 export class SetTitlePacket extends Packet {
-    // unknown
+    @nativeField(int32_t)
+    type:int32_t;
+    @nativeField(CxxString, 0x38)
```

Comment:
Thanks for your contribution
this point , CxxString is aligned to 8 bytes. 
so it will be 0x38 automatically not 0x34.